src/models/countries.js

```diff
@@ -0,0 +1,101 @@
+export default (sequelize, DataTypes) => {
+  const Country = sequelize.define("Country", {
+    nameOfCountry: {
+      type: DataTypes.ENUM,
+      values: [
```

Comment:
this should be an example of your ENUM e.g `DataTypes.ENUM('Super Admin', 'Travel Admin', 'Manager', 'Staff'),` not the values array you have there.

Reply:
done